tor_core/heartbeat.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+import cherrypy
+
+from tor_core.config import config
+
+conf = {
+    '/': {
+        'request.dispatch': cherrypy.dispatch.MethodDispatcher(),
+        'tools.response_headers.on': True,
+        'tools.response_headers.headers': [('Content-Type', 'application/json')],
+    }
+}
+
+
+def configure_heartbeat(config):
+    """
+    Sets up and starts the cherrypy server that operates the json api endpoint
+    for the heartbeat. Usage:
+
+    >>> import requests
+    >>> requests.get('http://localhost:{portnumber}').json()
+    returns:
+    {
+        'bot_name': 'this_is_an_awesome_bot',
+        'bot_version': '9001',
+        'core_version': '0.2.0'
+    }
```

Comment:
You know, I bet if we changed this around a bit we might eventually be able to use this for a doctest style of testing... 😁 

That's a bit out of scope for this PR though. Nice work! 👍